treeherder/model/data_cycling/signature_remover.py

```diff
@@ -1,19 +1,17 @@
 import logging
 from typing import List
 
+import taskcluster
 from django.conf import settings
 from django.db import transaction
 from taskcluster.exceptions import TaskclusterRestFailure
 
 from treeherder.perf.models import PerformanceSignature
-from treeherder.services.taskcluster import TaskclusterModel
+from ...perf.email import DeleteNotificationWriter
```

Comment:
Is the signature remover still working properly? The unit tests for it seem to be passing

Reply:
So I've just used a breakpoint for the `tests/model/cycle_data/test_perfherder_cycling.py::test_signature_remover` test, to closely follow the execution flow.
It's looking fine: the notification service is reached & called with the correct parameters.